packages/server/src/unstable-core-do-not-import/procedureBuilder.ts

```diff
@@ -73,12 +73,15 @@ type ProcedureResolver<
   _TMeta,
   TContextOverrides,
   TInputOut,
-  TOutputIn,
-  TOutputOut,
+  TOutputParserIn,
+  $Output,
```

Comment:
Was the consistency not beneficial before? TOutputParserIn is nice an explicit, any reason the Out needs to have a different scheme?

Reply:
I just like having the `$`-prefix when it's an inferred type

Reply:
![image](https://github.com/trpc/trpc/assets/8896153/0e1d79e9-8b3b-453e-ab75-38446d224527)